package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bnc-onboard",
-  "version": "1.37.2",
+  "version": "1.37.3",
```

Comment:
@yagopv Thank you for this PR!
Can you update the layout of your versioning to include a feature version and revert the onboard version  when you get a chance? 
`{prod_version}-{pre-release_version}` in this case if develop doesn't have a feature version `1.37.2-0.0.1`
Otherwise the feature version would be incremented 
This allows us to test in staging more easily and the update the project version when merging to master and releasing.
Thank you!

Reply:
Done!!

Reply:
thanks!!

Reply:
Super fast, thanks! Looks like there is a merge conflict. After develop is pulled and merged I will pull down and test and get this merged in - Thank you!!!

Reply:
Pulled and resolved. It was basically the version number